install-deps.sh

```diff
@@ -314,6 +314,10 @@ else
                 ensure_decent_gcc_on_ubuntu 9 bionic
                 [ ! $NO_BOOST_PKGS ] && install_boost_on_ubuntu bionic
                 ;;
+            *Disco*)
+                ensure_decent_gcc_on_ubuntu 8 disco
```

Comment:
@p-se i think we can simply drop this line once we have #30410 merged. as, on ubuntu Disco, we should have GCC-8. see https://packages.ubuntu.com/disco/gcc . so no need to install it from a PPA repo.

Reply:
Yes, that has worked with the changes in #30410. I have remove that line. Does that work for other Ubuntu versions, too, or do they require a particular GCC version?

Reply:
i think i tried to answer your question in the commit message in #30410.